SharePointFramework/PortfolioWebParts/src/components/ProjectList/ProjectCard/context.ts

```diff
@@ -1,24 +1,28 @@
-import { IButtonProps } from '@fluentui/react/lib/Button'
 import { createContext } from 'react'
 import { IProjectListProps } from '../types'
 import { ProjectListModel } from 'pp365-shared-library/lib/models'
+import { ButtonProps } from '@fluentui/react-components'
 
 export interface IProjectCardContext
-  extends Pick<IProjectListProps, 'showProjectLogo' | 'showProjectOwner' | 'showProjectManager'> {
+  extends Pick<
```

Comment:
Mulig vi burde ha egen interface for `IProjectCardContext` når Pick blir så kompleks?